modules/Wordlist/README.md

```diff
@@ -13,8 +13,8 @@ thus in CLDF only a [`FormTable`](../../components/forms) is required.
 Providing support for the handling of lexical data is one of the core goals of CLDF. By now, many examples on how to handle certain kinds of data have been presented in talks but also implemented in different tools, including [LingPy](http://lingpy.org) and [EDICTOR](http://edictor.digling.org). Lexical data is to a large extent much more amenable to strict cross-linguistic formalization than other types of linguistic data, but it is obvious that not all language-specific problems can be handled at once. A general idea behind CLDF is to not be intimidated by apparent complexity but instead to try to amplify the power of standards step by step.
 
 The framework for lexical specifications, appropriate for word list data, is roughly oriented toward the model of the linguistic sign (which can be a word but also a morpheme) which consists of a **language** in which the sign is expressed, a **meaning** that the sign expresses, and a **form** that denotates the meaning. 
-- Language specification should be done by linking to reference catalogs like Glottolog, but specifications of linguistic varieties may need to be more fine-grained than e.g. Glottolog allows at the moment. 
-- Meaning specifications again should use reference catalogs like Concepticon. This may not be possible, though, in very specific dialect datasets where many cultural terms are of interest when studying language variation.
+- Language specification should be done by linking to reference catalogs like [Glottolog](http://glottolog.org), but specifications of linguistic varieties may need to be more fine-grained than e.g. Glottolog allows at the moment. 
```

Comment:
maybe even more defensive: "ideally by linking to" rather than "should be done by linking to"?

Reply:
yes, thanks, I'll put that on the list of things.

Reply:
okay, addressed in my last push